ParisBNF/abb/BNFabb203.xml

```diff
@@ -92,18 +162,126 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                tree branches.</note>
+                           </decoNote>
+                        </decoDesc>
+
+                          <additions>
+                          <list>
+                          <item xml:id="a1">
+                            <locus target="#1r"/>
+                             <desc type="Unclear" xml:lang="fr"></desc>
+                             <note>Three sentence written by <persName ref="PRS9683Urbino">
+                               Giusto da Urbino</persName>.</note>
+                               <q xml:lang="fr">Ce n'est pas peu de chose que
+                                  d'être content de Dieu et de l'Univers.
+                                  Tous les peuples ont reconnu d'un commun accord qu'il
+                                  existe un Dieu dont la sagesse gouverne l'Univers.
+                                Le ciel pardonne tout hors d'inhumanité.</q>
+                                <listBibl type="secondary">
+                                   <bibl>
+                                      <ptr target="bm:Wion2013Hatata1"/>
+                                      <citedRange unit="page">48-49</citedRange>
+                                   </bibl>
+                                </listBibl>
+                             </item>
+
+                          <item xml:id="a2">
+                             <locus target="#1r"/>
+                              <desc type="Unclear" xml:lang="fr"></desc>
+                              <q xml:lang="fr">.</q>
+                              <note>The note identifies the preceeding note
+                                as penned by <persName ref="PRS9683Urbino">
+                                Giusto da Urbino</persName>.</note>
+                                <q xml:lang="fr">Ce qui précède est de l'écriture du R.P. Juste
+                                  d'Urbin mort à Kartum en Nubie, martyr de l'obéissance, le</q>
+                              </item>
+
+                              <item xml:id="e1">
+                                  <desc type="Unclear" xml:lang="gez">An unclear note on
+                                    the <ref target="#b5">left pastedown</ref>.</desc>
+                                <q xml:lang="gez">ድ᎓ ወጐንዮታ᎓ ያጤይቅ᎓ ጽድቀ᎓ ዚአሃ᎓</q>
+                                  </item>
 
-                        <additions>
-                      </additions>
+                                  <item xml:id="e2">
+                                     <desc>Stamp of the library,
+                                       on the <ref target="#b5">left pastedown</ref>.</desc>
+                                      </item>
+
+
+                                  <item xml:id="e3">
+                                     <locus target="#1r"/>
+                                      <desc type="OwnershipNote" xml:lang="gez"/>
+                                    <q xml:lang="gez">ዝመጽሐፍ᎓ ዘአባ᎓ ዮስጦስ᎓</q>
```

Comment:
the name could also be marked up in fidal

Reply:
Done.